src/components/DeleteMovieDialog/DeleteMovieDialog.scss

```
@@ -21,53 +21,56 @@
     left: 50%;
     transform: translate(-50%, -50%);
 
-    width: 686px;
-    height: 352px;
+    display: flex;
+    flex-direction: column;
+    row-gap: 30px;
+
+    width: 700px;
+    height: 400px;
     z-index: $z-index-modal;
 
     background: $primary-background-color;
     box-shadow: 0px 0px 2px rgba(0, 0, 0, 0.1), 0px 2px 10px rgba(0, 0, 0, 0.07),
       0px 10px 20px rgba(0, 0, 0, 0.05), 0px 10px 50px rgba(0, 0, 0, 0.05);
 
     .close-button {
-      position: absolute;
-      right: 26px;
-      top: 28px;
+      margin-right: 20px;
+      margin-top: 20px;
+      display: flex;
+      justify-content: flex-end;
 
       background-color: inherit;
       border: none;
     }
 
     h2 {
-      @include font($weight: 300, $size: 40px, $height: 49px);
+      @include font($weight: 300, $size: 40px, $height: 50px);
+
+      display: flex;
+      justify-content: space-evenly;
 
       letter-spacing: 2px;
       text-transform: uppercase;
-
-      margin-top: 64px;
-      margin-left: 82px;
-      margin-bottom: 52px;
     }
 
     p {
       @include font($weight: 400, $size: 20px, $height: 24px);
 
-      margin-left: 82px;
-      margin-top: 38px;
+      display: flex;
+      justify-content: space-evenly;
```

If your goal is to center the element, then you should try using [other rules](https://css-tricks.com/centering-css-complete-guide/) since the current implementation doesn't do that for two or more items, and certainly doesn't give that information to the other developers that will look at this declarations.

```suggestion
      margin: 0 auto;
```
OR

```suggestion
      display: flex;
      justify-content: center;
```

Alternatively, you could declare flex on the parent container and state that all children should be centered